### ISSUE TITLE
problem when synthesize

### ISSUE THREAD

**Issue by binyi10** [ksw0306/FloWaveNet#10]
thanks for this nice job
1. but I have some promble when **synthesize**, there is always sound reverberation in synthesize audio compared with raw audio, does someone have same problem with me(batch size = 4,1000k step). I guess the "change order"module may lead in this problem?

2. after 200k step, I found loss almost remain unchanged in (-3.4,-3.7), and the result of synthesize is similar too from 200k to 1000k，so I want to ask it is reasonable? and if not, which scale of **loss**  is reasonable
![image](https://user-images.githubusercontent.com/13240675/49005546-ee02f500-f1a1-11e8-9a11-a78254d2c0b4.png)

**Comment by jiqizaisikao** [comment]
Hi,can you upload your result wav

**Comment by binyi10** [comment]
raw: http://github.com/binyi10/-/blob/master/yibintest7.wav
syn: http://github.com/binyi10/-/blob/master/generate_890682_7_0.7.wav

>           Your wave maybe wrong and cannot be opened

sorry,my fault, you can copy these urls to your browser, or git clone http://github.com/binyi10/- to you pc, then you can listen the wave

**Comment by binyi10** [comment]
>           Hi,can you upload your result wav
hello，instead of using ljspeech data ,I use my dataset, so I change the source code about some para(e.g. change the value of hop_length = 120), the results have been uploaded, do you meet the similar problem?

**Comment by ksw0306** [comment]
How many audio samples are in your dataset? The scale of the loss function varies from dataset to dataset(VCTK, LJSpeech, ...). In the case of LJSpeech, it's near -4.5.

**Comment by jiqizaisikao** [comment]
Your wave maybe wrong and cannot be opened

**Comment by binyi10** [comment]
>           Your wave maybe wrong and cannot be opened
sorry, you can link to this repo[https://github.com/binyi10/-](url)
raw: yibintest7.wav
syn: generate_890682_7_0.7.wav

**Comment by binyi10** [comment]
>           Your wave maybe wrong and cannot be opened



>           Your wave maybe wrong and cannot be opened

sorry, 

>           How many audio samples are in your dataset? The scale of the loss function varies from dataset to dataset(VCTK, LJSpeech, ...). In the case of LJSpeech, it's near -4.5.

i use 30k samples of audio, but my cuda mem only support me use batch size = 4, so I think maybe it's suboptimal, I have try to change your code to multi-gpu, I use pytorch function **_DATAPARALLEL(model)_**, and I change your code in train.py in line 110 

`loss = -(log_p + logdet).mean()`

but the loss may be wrong, and I syn the audio is just noise. 
can you open a multi-gpu version code?

**Comment by binyi10** [comment]
>           How many audio samples are in your dataset? The scale of the loss function varies from dataset to dataset(VCTK, LJSpeech, ...). In the case of LJSpeech, it's near -4.5.

i use 30k samples of audio, but my cuda mem only support me use batch size = 4, so I think maybe it's suboptimal, I have try to change your code to multi-gpu, I use pytorch function **_DATAPARALLEL(model)_**, and I change your code in train.py in line 110 

`loss = -(log_p + logdet).mean()`

but the loss may be wrong, and I syn the audio is just noise. 
can you open a multi-gpu version code?

**Comment by L0SG** [comment]
Actually for the multi-GPU training there is subtle caveat. The `ActNorm` parameters need to be initialized with the first training batch _before_ converting the model into `DataParallel`, or else the training diverges badly. We'll upload the (correct) multi-GPU training part of the code shortly.

**Comment by L0SG** [comment]
[The latest commit](https://github.com/ksw0306/FloWaveNet/commit/a578151daccb31c44223876cee87633e9d234918) now supports the proper multi-GPU training.

**Comment by L0SG** [comment]
We'll close the issue, but feel free to re-open if the problem persists. FYI: please also refer to #13 .
